skfda/preprocessing/dim_reduction/_fpca.py

```diff
@@ -29,8 +29,10 @@ class FPCA(  # noqa: WPS230 (too many public attributes)
 
     Class that implements functional principal component analysis for both
     basis and grid representations of the data. The parameters are shared
-    when fitting a FDataBasis or FDataGrid, except for
-    ``components_basis``.
+    when fitting a FDataBasis or FDataGrid, except for ``components_basis``.
+
+    For more information about the implementation of the computation of the
+    first principal components see :footcite:ts:`silverman_2005_basisfuncexp`.
```

Comment:
The reference is wrong. Please, check the generated documentation (either locally, or in the PR, in the details link of the Readthedocs check).